packages/siwx/src/configs/DefaultSIWX.ts

```diff
@@ -11,7 +11,7 @@ const DEFAULTS = {
       getNonce: async () => Promise.resolve(Math.round(Math.random() * 10000).toString())
     }),
 
-  getDefaultVerifiers: () => [new EIP155Verifier()],
+  getDefaultVerifiers: () => [new EIP155Verifier(), new SolanaVerifier()],
```

Comment:
nit: by looking at this, wouldn't it be better to have `{ eip155: EIP155Verifier, solana: SolanaVerifier }`?

Reply:
My thought on this was to allow combine multiple verifiers in a same chain if for some reason that's needed. For example if validating signature and something in server side specific for the chain. Do you think this makes sense?